package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "codecov-action",
-  "version": "1.5.2",
+  "version": "2.0.0",
   "description": "Upload coverage reports to Codecov from GitHub Actions",
   "main": "index.js",
   "scripts": {
+    "build": "ncc build src/index.ts",
     "lint": "eslint src/*.*",
     "test": "npm run test-script && npm run test-calculator && npm run test-coverage",
-    "test-calculator": "jest --testPathPattern=demo/calculator/ --coverage --coverageDirectory=coverage/calculator",
+    "test-calculator": "jest --testPathPattern=demo/calculator/ --coverage --coverageDirector=coverage/calculator",
```

Comment:
```suggestion
    "test-calculator": "jest --testPathPattern=demo/calculator/ --coverage --coverageDirectory=coverage/calculator",
```

Reply:
Ahhh, that explains it. Nice catch @RA80533